blazor/datagrid/sorting.md

```diff
@@ -275,9 +275,10 @@ N> The SortComparer property will work only for local data.
 
 ## Touch interaction
 
-When you tap the datagrid header on touchscreen devices, the selected column header is sorted. A popup ![sorting](./images/sorting.jpg) is displayed for multi-column sorting. To sort multiple columns, tap the popup![msorting](./images/msorting.jpg), and then tap the desired datagrid headers.
+When you tap the datagrid header on touchscreen devices, the selected column header is sorted.To sort multiple columns, tap the desired datagrid headers.
 
-N> The [AllowMultiSorting](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowMultiSorting) and [AllowSorting](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowSorting) should be **true** then only the popup will be shown.
+N> The [AllowSorting](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowSorting) must be **true** while enabling multi-column sort.
```

Comment:
AllowSorting and AllowMultiSorting should be enabled to perform Multi sorting. not only AllowSorting property, this line leads confusion to users.

Reply:
Hi @Venkat-Ayothi ,

modified the content as per your suggestion.

Regards,
Adithyan